src/config/mod.rs

```diff
@@ -674,7 +674,7 @@ mod test {
 
     #[test]
     fn test_was_set() {
-        let config = Config::from_toml("hard_tabs = true", Path::new("")).unwrap();
+        let config = Config::from_toml("hard_tabs = true", Path::new(".")).unwrap();
```

Comment:
Let's change this to a file path to better convey that `from_toml` now expects a file path instead of a directory.

```rust
        let config = Config::from_toml("hard_tabs = true", Path::new("./rustfmt.toml")).unwrap();
```

Reply:
should i do this for every test that takes `from_toml` or just this?

Reply:
All of them

Reply:
updated , thank you